### DISCUSSION TITLE
Color braces of `$lr(..)$` in different colors.

### DISCUSSION THREAD

**Discussion by N4tus** [typst/typst#1122] [Q&A]
It is possible to color the two braces from an `$lr(..)$` element?  
Using `$ text(lr(( text(integral, fill: #blue) )), fill: #red)  $` I can color the entire `$lr(..)$` element in on color and the change the color for the content, but then both braces have the same color.   
I tried doing `$ lr(text("(", fill: #red) text(integral_(-infinity)^infinity, fill: #blue) text(")", fill: #green)) $` and the braces are rendered correctly except they are not colored.

**Comment by laurmaedje** [comment]
Fixed in #1195.

**Comment by sitandr** [comment]
That means that in new typst release your code with `text("(", fill: #red)` will hopefully work (there was a bug there that erased the local styles). I also added this example to tests, so it shouldn't break again.
